lib/core/src/display/scene/htmlrenderer.rs

```diff
@@ -0,0 +1,104 @@
+use super::{Camera, HTMLScene};
+
+fn eps(value: f32) -> f32 {
+    if value.abs() < 1e-10 {
+        0.0
+    } else {
+        value
+    }
+}
```

Comment:
Hehe, I love this function :D 
Make it shorter:
```rust
fn eps(value: f32) -> f32 {
    if value.abs() < 1e-10 { 0.0 } else { value }
}
```

Reply:
Damn you, rustfmt!